packages/chain-adapters/src/types.ts

```diff
@@ -9,9 +9,9 @@ import {
 import { BIP44Params, ChainSpecific, KnownChainIds, UtxoAccountType } from '@shapeshiftoss/types'
 
 import * as bitcoin from './bitcoin'
-import * as cosmos from './cosmos'
+import * as cosmos from './cosmossdk/cosmos'
+import * as osmosis from './cosmossdk/osmosis'
 import * as ethereum from './ethereum'
-import * as osmosis from './osmosis'
 
 export { bitcoin, cosmos, ethereum }
```

Comment:
```suggestion
```

Reply:
That would be a breaking change, so the PR would need to be updated to reflect this

Reply:
https://github.com/shapeshift/lib/pull/791/commits/eb85b288cc9bb392632a8f3d2ae92ff865f09791